src/keywords/data1dstore.cpp

```diff
@@ -45,8 +45,9 @@ bool Data1DStoreKeyword::deserialise(LineParser &parser, int startArg, const Cor
 // Serialise data to specified LineParser
 bool Data1DStoreKeyword::serialise(LineParser &parser, std::string_view keywordName, std::string_view prefix) const
 {
-    for (const auto &[data, format] : data_.data())
+    for (const auto sharedDataPointer : data_.data())
```

Comment:
Just a small thing, but we can avoid copying the shared pointer here and use a reference:
```suggestion
    for (const auto &sharedDataPointer : data_.data())
```